### PR TITLE
Hud10837/readme gotchas

### PR DESCRIPTION
**Pull request by hud10837**
This PR covers all of the readme gotchas from https://devtopia.esri.com/runtime/common-samples/issues/1823 that I could find which applied to Android! @TADraeseke if you could have a look when you get the chance, it should be pretty quick :)

### CODE REVIEW

File: java/map-load-status/README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ Tap on the refresh button in the top right of the screen to reload the ArcGISMap
 
 ## How it works
 
+1. Create an `ArcGISMap` and add it to a `MapView`.
+2. Use `ArcGISMap.addLoadStatusChangedListener()` or `ArcGISMap.getNewLoadStatus()` to 	query the map's load status.
```
Comment (TADraeseke) [inline java/map-load-status/README.md:14]:
```suggestion
2. Use `ArcGISMap.addLoadStatusChangedListener()` or `ArcGISMap.getNewLoadStatus()` to query the map's load status.
```

File: java/set-initial-map-area/README.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ Display the map at an initial viewpoint representing a bounding geometry.
 
 ## Use case
 
-Setting the initial viewpoint is useful when a user wishes to first load the map at a particular area of interest. 
+Setting the initial viewpoint is useful when a user wishes to first load the map at a particular area of interest.
 
 ## How to use the sample
 
-As the application is loading, the initial view point is set and the map view opens at the given location.
+When the sample loads, note the map is opened at the initial view point that is set to it.
```
Comment (TADraeseke) [inline java/set-initial-map-area/README.md:13]:
```suggestion
When the sample loads, note that the map is opened at the initial view point set.
```

**Comment by hud10837** [comment]
Thanks for those @TADraeseke , could you approve if it is looking good now?

File: java/control-annotation-sublayer-visibility/README.md [hunk 3]
```diff
@@ -18,14 +18,10 @@ Use the checkboxes to manually set "Open" and "Closed" annotation sublayers visi
...
 
-1. Load the `MobileMapPackage`.
-2. Populate checkbox text with the `AnnotationSublayer` names (including maximum and minimum scales, where relevant).
-3. Wire up the checkboxes to toggle the annotation sublayer's visibility.
-4. Add a listener for changes in map view navigation and add code to:
-  * Update the current scale UI element at the bottom of the screen on navigation.
-  * Set the "Open" annotation sublayer checkbox text to:
-     * Black, when the layer is drawn at the current map scale
-     * Light gray, when the layer is not drawn at the current map scale
+ 1. Load a `MobileMapPackage` that contains `AnnotationSublayer`.
```
Comment (TADraeseke) [inline java/control-annotation-sublayer-visibility/README.md:21]:
```suggestion
 1. Load a `MobileMapPackage` that contains an `AnnotationSublayer`.
```

**Comment by hud10837** [comment]
Thanks Trevor! Merging now

**Pull request merged**